src/components/data/row/CurrencyRow.tsx

```diff
@@ -10,7 +10,19 @@ import { FiatText } from '../../text/FiatText'
 import { TickerText } from '../../text/TickerText'
 import { IconDataRow } from './IconDataRow'
 
+// For display of custom assets such as AAVE collateral tokens
+export interface CustomAsset {
+  displayName: string
+  currencyCode: string
+  // TODO: Update after hidden assets are supported in accountbased
+  nativeBalance: string
+  // Token referenced for its exchange rate and icon
+  referenceTokenId: string
+  wallet: EdgeCurrencyWallet
+}
```

Comment:
This is ultimately an issue of accruing tech-debt with this `WalletListModal` being overloaded with other data type parameter which it accepts. A good solution in a later PR to fix this is to refactor `WalletListModal` to be generic in the asset type it expects and then make functions that transform wallets and other data types into the expected data type in order to search/list "asset-like" items. This would be more composable and flexible.